### PR TITLE
[TS SDK] Add Indexer GQL queries

### PR DESCRIPTION
**Pull request by 0xmaayan**
### Description
Adding Indexer queries Explorer uses so they can stop maintaining Apollo and GQL and utilize TS SDK. 
https://github.com/aptos-labs/aptos-core/issues/7076

### Test Plan
tests are passing

### CODE REVIEW

File: ecosystem/typescript/sdk/src/indexer/queries/getAccountCoinsData.graphql [hunk 1]
```diff
@@ -0,0 +1,11 @@
+query getAccountCoinsData($owner_address: String, $offset: Int, $limit: Int) {
```
Comment (jillxuu) [inline ecosystem/typescript/sdk/src/indexer/queries/getAccountCoinsData.graphql:1]: noob q: are these graphql queries used anywhere? why we need to have these queries if they are already exported from `queries.ts`?

**Review by jillxuu — APPROVED** [review]

File: ecosystem/typescript/sdk/src/indexer/queries/getAccountCoinsData.graphql (hunk 1 above)
Comment (0xmaayan) [reply ecosystem/typescript/sdk/src/indexer/queries/getAccountCoinsData.graphql:1]: `queries.ts` is generated based on the queries in `.graphql` files. so write a gql query in a `.graphql` file, then generate those queries into a `.ts` file and use it.

File: ecosystem/typescript/sdk/src/providers/indexer.ts [hunk 2]
```diff
@@ -79,10 +126,187 @@ export class IndexerClient {
...
+    IndexerClient.validateAddress(address);
+    const graphqlQuery = {
+      query: GetAccountTokensCount,
+      variables: { owner_address: address },
+    };
+    return this.queryIndexer(graphqlQuery);
+  }
+
+  /**
+   * Queries an account transactions acount
```
Comment (banool) [inline ecosystem/typescript/sdk/src/providers/indexer.ts:170]:
```suggestion
   * Queries an account transactions count
```

Same thing for other comments, `acount` -> `count`.

Comment (banool) [reply ecosystem/typescript/sdk/src/providers/indexer.ts:170]: Though something like "Gets the count of transactions submitted by an account" would be more descriptive.

**Review by banool — CHANGES_REQUESTED** [review]

**Review by banool — APPROVED** [review]

**Pull request merged**